GitUpKit/Utilities/GIViewController+Utilities.m

```diff
@@ -571,6 +583,15 @@ - (void)resolveConflictInMergeTool:(GCIndexConflict*)conflict {
     [self _runP4MergeWithArguments:arguments];
   } else if ([identifier isEqualToString:GIViewControllerTool_GitTool]) {
     [self _runMergeGitToolForFile:mergePath withOldPath:ourPath newPath:theirPath basePath:ancestorPath];
+  } else if ([identifier isEqualToString:GIViewControllerTool_DiffMerge]) {
+    [arguments addObject:[NSString stringWithFormat:@"-r=%@", mergePath]];
+    [arguments addObject:[NSString stringWithFormat:@"-t1=%@", ourTitle]];
+    [arguments addObject:[NSString stringWithFormat:@"-t2=%@", ancestorTitle]];
+    [arguments addObject:[NSString stringWithFormat:@"-t3=%@", theirTitle]];
+    [arguments addObject: ourPath];
+    [arguments addObject: ancestorPath];
+    [arguments addObject: theirPath];
+    [self _runDiffMergeToolWithArguments: arguments];
```

Comment:
Could we remove the spacing after the colon in these 4 spots. Other than that, code wise this looks good to me.